testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/transaction/MaximumTimeoutTestCase.java

```diff
@@ -0,0 +1,213 @@
+package org.jboss.as.test.integration.transaction;
```

Comment:
I don't think this is too much of a big deal now, but this should probably have a copyright header.

Reply:
I just added it now thanks